recipe/meta.yaml

```diff
@@ -26,13 +26,20 @@ source:
     folder: third_party/googletest
 
 build:
-  number: 1
+  number: 0
+
+requirements:
+  build:
+    - patch  # [not win]
```

Comment:
I guess that here m2-patch fow Windows is missing.  As this recipe seems to work on Windows too, as there is no skip for it

Reply:
hmm, maybe not ... those patchees are not applied for Windows ...  So add to the 'patches:' line the jinja2 preprocessor statement '[ppc64le or aarch64 or s390x]'. So we avoid and empty yaml entry

Reply:
Made entries more specific.